opencti-platform/opencti-front/src/private/components/data/ingestionCsv/IngestionCsvCreation.tsx

```diff
@@ -279,6 +281,18 @@ const IngestionCsvCreation: FunctionComponent<IngestionCsvCreationProps> = ({ pa
                 isOptionEqualToValue={(option: Option, value: string) => option.value === value}
                 containerStyle={fieldSpacingContainerStyle}
               />
+              <Box>
+                <div style={{ width: '100%', marginTop: 30 }}>
```

Comment:
I think you can merge these 2 tags using the Box `sx` prop. Something like:
`<Box sx={{ width: '100%', marginTop: 30 }}>`

We try to avoid inline style as much as possible.